app/templates/deliveries/new/confirm.hbs

```diff
@@ -0,0 +1,21 @@
+{{delivery-breadcrumbs selectedRouteName="deliveries.confirm"}}
+<h1>New Delivery</h1>
+<h4>
+  Please confirm the delivery of <b>{{project.name}}</b> below.
```

Comment:
No such property `project.name`. Shows up as `Please confirm the delivery of below.` in UI.

Reply:
Since the project name should appear in the preview and I'm in the process of refactoring, I'm just going to remove the `{{project.name}}` variable from this line.